compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/tile_and_fuse_using_scf_forall.mlir

```diff
@@ -63,6 +80,55 @@ func.func @matmul_static(%arg0: tensor<8x16xi32>, %arg1 : tensor<16x8xi32>) -> t
+    %extracted_slice_0 = tensor.extract_slice %arg1[%iv0, 0, 0, 0] [4, 32, 3, 3] [1, 1, 1, 1] : tensor<64x32x3x3xf32> to tensor<4x32x3x3xf32>
+    %extracted_slice_1 = tensor.extract_slice %arg3[0, %iv0, %iv1, %iv2] [2, 4, 4, 4] [1, 1, 1, 1] : tensor<2x64x12x12xf32> to tensor<2x4x4x4xf32>
+    %7 = linalg.fill ins(%cst : f32) outs(%extracted_slice_1 : tensor<2x4x4x4xf32>) -> tensor<2x4x4x4xf32>
+    %cst_2 = arith.constant 0.000000e+00 : f32
+    %8 = bufferization.alloc_tensor() : tensor<2x32x6x6xf32>
+    %alloc = memref.alloc() : memref<2x32x6x6xf32, 1 : i32>
+    %9 = bufferization.to_tensor %alloc restrict writable : memref<2x32x6x6xf32, 1 : i32>
+    %10 = linalg.copy ins(%extracted_slice : tensor<2x32x6x6xf32>) outs(%9 : tensor<2x32x6x6xf32>) -> tensor<2x32x6x6xf32>
+    %cst_3 = arith.constant 0.000000e+00 : f32
+    %11 = bufferization.alloc_tensor() : tensor<4x32x3x3xf32>
+    %alloc_4 = memref.alloc() : memref<4x32x3x3xf32, 1 : i32>
+    %12 = bufferization.to_tensor %alloc_4 restrict writable : memref<4x32x3x3xf32, 1 : i32>
+    %13 = linalg.copy ins(%extracted_slice_0 : tensor<4x32x3x3xf32>) outs(%12 : tensor<4x32x3x3xf32>) -> tensor<4x32x3x3xf32>
+    %cst_5 = arith.constant 0.000000e+00 : f32
+    %14 = bufferization.alloc_tensor() : tensor<2x4x4x4xf32>
+    %alloc_6 = memref.alloc() : memref<2x4x4x4xf32, 1 : i32>
+    %15 = bufferization.to_tensor %alloc_6 restrict writable : memref<2x4x4x4xf32, 1 : i32>
+    %16 = linalg.copy ins(%7 : tensor<2x4x4x4xf32>) outs(%15 : tensor<2x4x4x4xf32>) -> tensor<2x4x4x4xf32>
+    %17 = linalg.conv_2d_nchw_fchw {dilations = dense<1> : vector<2xi64>, lowering_config = #iree_codegen.lowering_config<tile_sizes = [[0, 4, 4, 4, 0, 0, 0], [1, 4, 1, 4, 0, 0, 0], [0, 0, 0, 0, 8, 1, 1]]>, strides = dense<1> : vector<2xi64>} ins(%10, %13 : tensor<2x32x6x6xf32>, tensor<4x32x3x3xf32>) outs(%16 : tensor<2x4x4x4xf32>) -> tensor<2x4x4x4xf32>
+    %extracted_slice_7 = tensor.extract_slice %17[0, 0, 0, 0] [2, 4, 4, 4] [1, 1, 1, 1] : tensor<2x4x4x4xf32> to tensor<2x4x4x4xf32>
+    %18 = linalg.copy ins(%extracted_slice_7 : tensor<2x4x4x4xf32>) outs(%7 : tensor<2x4x4x4xf32>) -> tensor<2x4x4x4xf32>
+    memref.dealloc %alloc : memref<2x32x6x6xf32, 1 : i32>
+    memref.dealloc %alloc_4 : memref<4x32x3x3xf32, 1 : i32>
+    memref.dealloc %alloc_6 : memref<2x4x4x4xf32, 1 : i32>
+    scf.forall.in_parallel {
+      tensor.parallel_insert_slice %18 into %arg3[0, %iv0, %iv1, %iv2] [2, 4, 4, 4] [1, 1, 1, 1] : tensor<2x4x4x4xf32> into tensor<2x64x12x12xf32>
+    }
+  }
+  return %6 : tensor<2x64x12x12xf32>
+}
+
+//      TILE-LEVEL-1: @conv_2d_nchw_fchw
+//      TILE-LEVEL-1:   scf.forall
+// TILE-LEVEL-1-SAME:   {
+//      TILE-LEVEL-1:       linalg.fill
+//      TILE-LEVEL-1:       scf.forall
+// TILE-LEVEL-1-SAME:       {
+//      TILE-LEVEL-1:           linalg.conv_2d_nchw_fchw
+//      TILE-LEVEL-1:       }
```

Comment:
@MaheshRavishankar @Abhishek-Varma Any idea why `linalg.fill` is not fused into the forall loop after tiling for this Conv2d test? I think it worked for matmul tests. This is not a critical issue at the moment, since I'm using a separate pass `createAMDAIEFuseFillIntoForallPass()` for fusion.

Reply:
My guess would be the linalg.copy that is in the input IR, can you check without it?